dev/io.openliberty.mcp.internal/resources/io/openliberty/mcp/internal/CWMCM.nlsprops

```diff
@@ -0,0 +1,153 @@
+
+# Used for server logging
+# do not translate MCP
+# {0} = The qualified name of the tool
+CWMCM0001E.blank.arguments=CWMCM0001E: The {0} MCP tool method has one or more arguments with blank names
+CWMCM0001E.blank.arguments.explanation=MCP Tool arguments were provided but they had missing names. Each argument to a tool method must have a name.
+CWMCM0001E.blank.arguments.useraction=Provide non-empty values for all required arguments.
+
+# Used for server logging
+# do not translate MCP
+# {0} = The  qualified name of the tool
+# {1} = corresponding argument name
+CWMCM0002E.duplicate.arguments=CWMCM0002E: The {0} MCP tool method has more than one argument named {1}
+CWMCM0002E.duplicate.arguments.explanation=One or more arguments have the same name. Each argument of an MCP tool method must have a different name.
+CWMCM0002E.duplicate.arguments.useraction=Give each tool argument a different name.
+
+# Used for server logging
+# do not translate MCP
+# {0} = The qualified name of the tool
+CWMCM0003E.missing.tool.argument.name=CWMCM0003E:  The {0} MCP tool method has one or more arguments without a name specified
+CWMCM0003E.missing.tool.argument.name.explanation=One or more Tool argument name(s) was not provided for MCP Tools.
+CWMCM0003E.missing.tool.argument.name.useraction=Either add a name to the @ToolArg annotation, or to add the -parameters compiler option to use the parameter name.
+
+# Used for server logging
+# do not translate MCP
+# {0} = The list of qualified names of the tool
+CWMCM0004E.duplicate.tools=CWMCM0004E: There are multiple MCP tool methods named {0}. The methods are {1}.
+CWMCM0004E.duplicate.tools.explanation=One or more MCP Tool names occured multiples times.The server expects one name per tool for uniqueness.
+CWMCM0004E.duplicate.tools.useraction=Only provide a unique name for each MCP Tool.
+
+# Used for server logging
+# do not translate MCP
+CWMCM0005E.validation.error=CWMCM0005E: There are one or more MCP validation errors. Check logs for more details.
+CWMCM0005E.validation.error.explanation=Server failed due to invalid arguments being passed.
+CWMCM0005E.validation.error.useraction=Check logs for more details.
+
+# Used for server logging
+# do not translate MCP
+# {0} = The fully qualified method name
+# {1} = The parameter type simple name
+CWMCM0006E.duplicate.special.arguments=CWMCM0006E: The {0} MCP Tool has more than one parameter with type {1}. There may only be one {1} parameter for each Tool method.
+CWMCM0006E.duplicate.special.arguments.explanation=One or more special arguments occured multiples times, only one is expected.
+CWMCM0006E.duplicate.special.arguments.useraction=Remove the extra instances.
+
+# Used for server logging
+# {0} = The fully qualified method name
+# {1} = The parameter type simple name
+CWMCM0007E.invalid.arguments=CWMCM0007E: The {0} MCP Tool has a parameter of type {1} which is not a recognised special argument type and does not have a `@ToolArg` annotation.
```

Comment:
Change "recognised" to "recognized" (US English spelling)

Make the same change to other uses of "recognised" in these messages.